src/components/small-print.js

```diff
@@ -10,15 +10,19 @@ var Footer = React.createClass({
     var wireTransferLink = (<Link to={'/' + this.context.intl.locale + '/ways-to-give#wire'}>{this.context.intl.formatMessage({id: 'wireTransfer'})}</Link>);
     var checkLink = (<a href={'/' + this.context.intl.locale + '/ways-to-give#check'}>{this.context.intl.formatMessage({id: 'check'})}</a>);
     var privacyPolicyMessage = "privacy_policy_var_b";
+    var refundsMessage = "";
     if (this.props.frequency === "monthly") {
       privacyPolicyMessage = "privacy_policy_var_b_monthly";
+      refundsMessage = this.context.intl.formatMessage({id: 'privacy_policy_var_b_monthly_refund'});
     }
 
     return (
       <div className="row disclaimers">
         <p className="full">
           <FormattedHTMLMessage id={privacyPolicyMessage}/>
           {` `}
+          <span>{refundsMessage}</span>
+          {` `}
```

Comment:
If we can't merge `privacy_policy_var_b_monthly_refund` into `privacy_policy_var_b_monthly`, then let's change

```
<span>{refundsMessage}</span>
{` `}
```
to

```
<span>{refundsMessage} </span>
```

so when `refundsMessage` is empty we don't end up having 2 spaces in between the 2 `<FormattedHTMLMessage ...>`